doc/screed.rst

```diff
@@ -1,181 +1,162 @@
-===================
-Basic Documentation
-===================
-
-.. contents:
+===========
+User Manual
+===========
 
-Notes on this document
-======================
-This is the default documentation for screed. Some doctests are included
-in the file 'example.txt'. The examples in this file are meant for humans
-only: they will not work in doctests.
+.. note::
 
-Introduction
-============
+   Some doctests are included in :doc:`example`. The examples in this
+   document are meant for human consumption only. They will not work in
+   doctests!
 
-screed parses FASTA and FASTQ files, generates databases, and lets you
-query these databases.  Values such as sequence name, sequence
-description, sequence quality, and the sequence itself can be
-retrieved from these databases.
+screed parses FASTA and FASTQ files, generates databases, and lets you query
+these databases. Values such as sequence name, sequence description, sequence
+quality, and the sequence itself can be retrieved from these databases.
 
-Getting Going
-=============
+Installation
+============
 
 The following software packages are required to run screed:
 
-* Python 2.4 or newer
-* pytest (for testing)
-
-Installing
------------
+* Python 2 (2.7) or Python 3 (3.3 or newer)
+* pytest (only required to running tests)
 
 Use pip to download, and install Screed and its dependencies::
 
     pip install screed
 
 To run the optional tests type::
 
-    nosetests screed --attr '!known_failing'
+    pip install pytest-cov
+    python -m screed.tests
 
-Quick-Start
-===========
+Command-line Quick Start
+========================
+
+Creating a database
+-------------------
+
+.. code::
+
+    $ screed db <fasta/fastq file>
+
+Dumping a database to a file
+----------------------------
+
+.. code::
+
+    $ screed dump_to_fasta <screed database file> <fasta output>
+    $ screed dump_to_fastq <screed database file> <fastq output>
+
+Python Quick Start
+==================
 
 Reading FASTA/FASTQ files
 -------------------------
 
-At the Python prompt, type::
-
    >>> import screed
    >>> with screed.open(filename) as seqfile:
    >>>     for read in seqfile:
-   ...         print read.name, read.sequence
+   ...         print(read.name, read.sequence)
 
-Here, 'filename' can be a FASTA or FASTQ file, and can be
-uncompressed, gzipped, or bz2-zipped.
+Here, :code:`filename` can be a FASTA or FASTQ file, and can be uncompressed,
+gzip-compressed, or bzip2-compressed. screed natively supports FASTA and FASTQ
+databases creation. If your sequences are in a different format see the
+developer documentation on :doc:`dev/parsers`.
 
-Creating a database from the API
---------------------------------
-
-From a Python prompt type::
+Creating a database
+-------------------
```

Comment:
@luizirber Updated. Note, currently `import screed; screed.make_db(file)` doesn't work. It looks like you're addressing that in #63, but for now I have `import screed; screed.createscreed.make_db(file)` as that is what works.